numpy/linalg/setup.py

```diff
@@ -59,11 +59,17 @@ def get_lapack_lite_sources(ext, build_dir):
                 return []
             return [all_sources[0]]
 
+    # AIX needs to be told to not use the extra underscore '_'
+    # Note: this setting is actually dependent on the fortran compiler behavior
+    # used to build liblapack.a (xlf does not append!, maybe gfortran does)
+    # Better way to establish liblapack.a definition is needed.
+    defs = [('NO_APPEND_FORTRAN', None)] if sys.platform[:3] == "aix" else []
```

Comment:
If it's compiler-dependent it should go into `IBMFCompiler` in `numpy/distutils/fcompiler/ibm.py` instead I think.

Reply:
Not the correct approach - this is a flag for the C compiler - to interface with fortran (libraries). IMHO - it will be a real nightmare if the libraries mix behaviors.

In any case, the AIX provided libraries do not have an underscore (_) appended, but user libraries might.

I would also like to suggest a README.AIX, if not already available, and/or I a link to a project I am starting for CPython -- https://github.com/aixtools/pythonAIXDOC -- there is no reason to limit it to CPython - as I am trying to be the focal point for AIX and OSS porting.

Not pressing "resolved", just yet.